wp-content/mu-plugins/query-vars.php

```diff
@@ -44,7 +44,7 @@
    array_push($vars, $prefix . 'dur'); // duration
    array_push($vars, $prefix . 'loc'); // locations
    array_push($vars, $prefix . 'pop'); // populations
-   array_push($vars, $prefix . 'age'); // age
+   array_push($vars, $prefix . 'ars'); // age_ranges_served
```

Comment:
```suggestion
   array_push($vars, $prefix . 'age'); // age_ranges_served
```

This needs to be consistent with archive.php and programs-archive.js